polyfills/Number/MAX_SAFE_INTEGER/config.json

```diff
@@ -0,0 +1,22 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "<12",
+		"ie_mob" : "*",
+		"firefox": "<31",
+		"firefox_mob":"<32",
+		"chrome":"<34",
+		"android": "<5",
+		"safari": "<9",
+		"ios_saf": "<9",
+		"opera": "*",
+		"op_mini": "*",
+		"op_mob": "*",
+		"bb": "*"
```

Comment:
We support Samsung 4 & above -- https://cdn.polyfill.io/v2/docs/contributing/authoring-polyfills#browsers